Documentation/help_files/openemr_installation_help.php

```diff
@@ -293,7 +294,59 @@
+                    
+                    <p><?php echo ("This key is generated by the setup script and is presented to the user in the form of a QR code");?>.
+                    
+                    <p><?php echo ("The QR code also contains the user name and needs to be captured by the user on to their mobile device using an authenticator app");?>.
+                    
+                    <p><?php echo ("Once this is done the shared secret key that is unique for each user should only exist in OpenEMR and on the user's authenticator app");?>.
+                    
+                    <p><?php echo ("Authenticator apps are available for both ios and android devices at their respective app stores and is free to use");?>.
+                    
+                    <p><?php echo ("The basic purpose of the authenticator app is to generate a 20 bytes (160 bits) code encoded in base32 using a secure hash function, SHA-1, and is called HMAC-SHA1 (Hash-based Message Authentication Code)");?>.
+                    
+                    <p><?php echo ("It does so by using an algorithm to combine the current UNIX time with the secret key to generate an ever changing unique key");?>.
+                    
+                    <p><?php echo ("For ease of use it is rendered as a unique 6 digit number");?>.
+                    
+                    <p><?php echo ("All TOTP authenticator apps use the same algorithm and secure hash function - HMAC-SHA1 to generate the unique key");?>.
+                    
+                    <p><?php echo ("An important concept to grasp is that once the secret key delivered via the QR code is captured by the authenticator app there is no further communication between the app and OpenEMR, each will use the current UNIX time and their copy of the user-specific shared secret key to generate the same unique 6 digit number");?>.
+                    
+                    <p><?php echo ("These numbers need to match to successfully authenticate the user");?>.
+                    
+                    <p><?php echo ("Once this feature is enabled you will be required to type in the 6 digit key at each login");?>.
+                    
+                    <p><?php echo ("After login you can choose to clear the key by deleting it under Miscellaneous > MFA Authentication");?>.
+                    
+                    <p><?php echo ("Those with administrator privileges can also go to Administration > Users and check Clear 2FA checkbox and Click Save");?>.
+                    
+                    <p><?php echo ("After login you can choose to add U2F key authentication or change to using U2F key authentication exclusively");?>.
+                    
+                    <p><?php echo ("A more detailed explanation is available in the help file for Miscellaneous > MFA Authentication");?>.
+                    
                     <p><?php echo ("Click on 'Create DB and User' button to continue");?>.
```

Comment:
In this script, remove the xla() calls on lines 759, 760, 776, and 777
(unable to translate on this script since there is no database yet)